engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -651,7 +709,13 @@ yield switch (fn) {
         }
       }
       case Tree.TypeAnnotated anno -> translateTypeAnnotated(anno);
-      case Tree.Group group -> translateExpression(group.getBody(), false);
+      case Tree.Group group -> {
+          yield switch (translateExpression(group.getBody(), false)) {
+              case null -> new IR$Error$Syntax(getIdentifiedLocation(group).get(), IR$Error$Syntax$EmptyParentheses$.MODULE$, meta(), diag());
+              case IR$Application$Prefix pref -> pref.setLocation(getIdentifiedLocation(group, 1, -1));
```

Comment:
can you add a comment on the magic `1, -1` constants?

Reply:
OK. Done in 081c79700